docs/photutils/psf.rst

```diff
@@ -481,6 +481,159 @@ Consider the previous example after the line
+    from photutils import MMMBackground
+    from photutils.psf import IntegratedGaussianPRF, DAOGroup
+    from photutils.detection import DAOStarFinder
+    from photutils.detection import IRAFStarFinder
+    from astropy.table import Table
+    from astropy.modeling.fitting import LevMarLSQFitter
+
+    sources = Table()
+    sources['flux'] = [10000, 1000]
+    sources['x_mean'] = [18, 13]
+    sources['y_mean'] = [17, 19]
+    sources['x_stddev'] = [2] * 2
+    sources['y_stddev'] = sources['x_stddev']
+    sources['theta'] = [0] * 2
+    tshape = (32, 32)
+    image = (make_gaussian_sources(tshape, sources) +
+             make_noise_image(tshape, type='poisson', mean=6.,
+                              random_state=1) +
+             make_noise_image(tshape, type='gaussian', mean=0.,
+                              stddev=2., random_state=1))
+
+    plt.imshow(image, cmap='viridis', aspect=1,
+               interpolation='nearest', origin='lower')
+
+Let's instantiate the necessary objetcs in order to use an
+`~photutils.psf.IterativelySubtractedPSFPhotometry` to perform photometry:
+
+.. doctest-requires:: scipy
+
+    >>> daogroup = DAOGroup(crit_separation=8)
+    >>> mmm_bkg = MMMBackground()
+    >>> iraffind = IRAFStarFinder(threshold=2.5*mmm_bkg(image), fwhm=4.5)
+    >>> fitter = LevMarLSQFitter()
+    >>> gaussian_prf = IntegratedGaussianPRF(sigma=2.05)
+    >>> gaussian_prf.sigma.fixed = False
+    >>> itr_phot_obj = IterativelySubtractedPSFPhotometry(finder=iraffind,
+    ...                                                   group_maker=daogroup,
+    ...                                                   bkg_estimator=mmm_bkg,
+    ...                                                   psf_model=psf_model,
+    ...                                                   fitter=fitter,
+    ...                                                   fitshape=(11, 11),
+    ...                                                   niters=None)
+
+Now, let's use the callable ``itr_phot_obj`` to perform photometry:
+
+.. doctest-requires:: scipy
+
+    >>> phot_results = itr_phot_obj(image)
+    >>> phot_results_itr['id', 'group_id', 'iter_detected', 'x_0', 'y_0', 'flux_0'] #doctest: +SKIP
+        id group_id iter_detected      x_0           y_0          flux_0
+        --- -------- ------------- ------------- ------------- -------------
+          1        1             1 17.9830836988  17.014907321 9753.12898388
+          1        1             2 12.6296930472 19.6647630336 722.116920665
+    >>> phot_results_itr['sigma_0', 'sigma_fit', 'x_fit', 'y_fit', 'flux_fit'] #doctest: +SKIP
+        sigma_0   sigma_fit       x_fit         y_fit        flux_fit
+        ------- ------------- ------------- ------------- -------------
+           2.05 2.03898434182 17.9155310586 17.0377322599 10478.1845716
+           2.05 1.77365765325 12.2809666868 19.2416219146 772.882624378
```

Comment:
I don't quite understand this.  It looks like the stars were created with the *same* size, yet the fit gives *different* sizes.  That would seem to be an *incorrect* fit.  

Maybe it would be better to have the second star be a bit further away, but be a bit closer in brightness and a different gaussian size?  That way you can actually end this example with "look, it correctly fit the different sizes"